### ISSUE TITLE
Find unnecessary usages of `asReversed`

### ISSUE THREAD

**Issue by BraisGabin** [detekt/detekt#7480]
## Expected Behavior of the rule
Find code like this:

```kolin
        .sortedBy { it.priority }
        .asReversed()
```

and propouse to use:
```kotlin
        .sortedByDescending { it.priority }
```

## Context
I found this problem in detekt's code: #7479

**Comment by atulgpt** [comment]
Your suggested code should be 
```
.sortedByDescending { it.priority }
```

**Comment by BraisGabin** [comment]
🤦 Good catch.

**Comment by psuzn** [comment]
Do we have any existing ruleset that can be enhanced or do we need to introduce a new one?

**Comment by BraisGabin** [comment]
We need a new one, I think.

**Comment by cortinico** [comment]
> We need a new one, I think.

Perhaps this could live inside the `style` ruleset where we have already rules like `UseLet` and so on?

**Comment by BraisGabin** [comment]
We need a new one, I think. Oh sure! I miss read it! A new rule, sure. Ruleset style.
